NashTeamAlpha/CustomerIncidentPortal
# Gr employee factory

## Pull Request Description

Include the applicable information from the list below here 

1. Added testing to getting an employee by their name.
1. Implemented method in the customer factory to get an employee by their name. 
1. Created Choose Existing Customer Action.
1. The Command Line Application will now accept a first and last name to login the employee
1. It has error handling and can handle multiple customers of the same name.
1. After a active customer has been selected the page should redirect to the MainMenu Action(In Progress)
1. Added a Banner Action that console writes the companies banner wherever it is invoked. 

## Steps to Test

1. Git Fetch
1. Git checkout gr-EmployeeFactory
1. Run the program
1. Try to break it
1. Check that tests and methods are functional. 

## Link to Feature Ticket

#30, #35, #2, #23
